openwisp_controller/config/migrations/0016_default_organization_config_settings.py

```diff
@@ -0,0 +1,30 @@
+from django.db import migrations
+
+
+def make_default_config_settings_organization(apps, schema_editor):
+    organization_model = apps.get_model('openwisp_users', 'Organization')
+    config_settings_model = apps.get_model(
+        'config',
+        'OrganizationConfigSettings'
+    )
+    for organization in organization_model.objects.all():
+        try:
+            config_setting = organization.config_settings
+        except organization_model.config_settings.RelatedObjectDoesNotExist:
+            # If there is no OrganizationConfigSettings associated with said organization
+            config_setting = config_settings_model.objects.create(organization=organization)
+            # creates OrganizationConfigSettings object corresponding to the organization

```

Comment:
suggestion for next time: put all comments related to a line or a block before, not after (eg: this comment should have been above line `15`)

Reply:
Will keep that in mind from now on.